python/ray/tune/examples/hyperopt_example.py

```diff
@@ -40,12 +40,12 @@ def easy_objective(config):
         {
             "width": 1,
             "height": 2,
-            "activation": 0  # Activation will be relu
+            "activation": "relu"  # Activation will be relu
```

Comment:
will it be possible to maintain backwards compat for a release or so?

Reply:
or at least have a really good error message about this -- as ray becomes more of an infra layer, subtle changes like this can really be a showstopper for migrations/upgrading.

Reply:
I guess the current error message is pretty good as-is. Will leave a small comment for improving the error message.